src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+    private final String color = ColorSupplier.getRandomColor();
+    private final int number = 100;
+    private int index = random.nextInt(number);
```

Comment:
only  `number` should be constant, index should be recalculated inside method, the code you have now means you will always return same index, generated when a Lottery object was created, that defeats purpose of this task

Reply:
intellij asks to do random final
<img width="1276" alt="Screenshot 2023-01-20 at 15 30 40" src="https://user-images.githubusercontent.com/122523652/213707604-05f9c36a-ab8a-48a0-a31e-cb9831653ccc.png">

Reply:
Read and analyze this: [Constants are static final fields whose contents are deeply immutable and whose methods have no detectable side effects](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)